src/org/camunda/latera/bss/connectors/hid/hydra/Customer.groovy

```diff
@@ -246,7 +246,7 @@ trait Customer {
         vch_VC_CODE           : params.code,
         vch_VC_REM            : params.rem
       ]
-      if (this.version > new Version('5') && notEmpty(params.resellerId)) {
```

Comment:
You can remove this check and add num_N_RESELLER_ID to args, because this check was designed to save backward compatibility with 4.1.3 there is no customer reseller id.